luminoth/train.py

```diff
@@ -142,18 +141,22 @@ def run(custom_config, model_type, override_params, target='',
         )
     else:
         checkpoint_dir = config.train.job_dir
-
-    if config.train.display_every_steps or config.train.display_every_secs:
-        if not config.train.debug:
+    if (config.train.display_every_steps or config.train.display_every_secs and
+            'image_vis' in config.train.keys()):
```

Comment:
Shouldn't we check that `image_vis` is not `None`? If `image_vis` is `None` in the `config.train` dict then the key will be in `.keys()`.

Reply:
Yes i dind't conider that, I'll modify it